### ISSUE TITLE
[Bug] IDE steals focus on tooltip

### ISSUE THREAD

**Issue by m910q** [beefytech/Beef#2019]
IDE steals focus from other applications when mouse hovering code that triggers a tooltip.

The bug only starts happening, once a debugging session has been started with the new "Debug Console" set to "Embedded".
When stopping debugging, it keeps happening.

Using the "Terminal" tab in the IDE does not trigger this bug.

**Comment by bfiete** [comment]
I haven't been able to reproduce. Maybe I'm missing a step or not understanding the reproduction instructions.

**Comment by m910q** [comment]
I just did some further testing, and found out it that the bug does indeed not always trigger on any debug session.

If I debug a project with a breakpoint that is hit, and continue from that, the bug does not trigger.
But if I debug a project that exits without breakpoints, it starts happening.

Release/Debug build does not matter.

Steps to reproduce:
- Open Beef IDE
- In Settings, set "Debug Console" to "Embedded" if not already
- Open random project (Works on a newly created empty project)
- Make sure no breakpoint is set
- Press F5 to debug and wait for it to end
Now the bug should be active

- Activate another application while Beef IDE is still visible in the background
- Mouseover a token in the code that produce a tooltip in Beef IDE
Beef IDE takes focus from the other application

I have tested on both Win10 and Win11.
